src/pages/home/ReportScreen.tsx

```diff
@@ -386,7 +386,13 @@ function ReportScreen({route, navigation}: ReportScreenProps) {
         () => !!linkedAction && !shouldReportActionBeVisible(linkedAction, linkedAction.reportActionID, canUserPerformWriteAction(report)),
         [linkedAction, report],
     );
-    const prevIsLinkedActionDeleted = usePrevious(linkedAction ? isLinkedActionDeleted : undefined);
+
+    const previsLinkedActionDeleted = usePrevious(linkedAction ? isLinkedActionDeleted : undefined);
+
+    const lastReportActionIDFromRoute = usePrevious(reportActionIDFromRoute);
+
+    const [isNavigatingToAction, setIsNavigatingToAction] = useState(false);
```

Comment:
`isNavigatingToAction` sounds more broad than it really is I think? it's only true if we're navigating to a _deleted_ action. So I think a more accurate variable name would be `isNavigatingToDeletedAction`